src/main/java/com/bravo/user/dao/model/Payment.java

```diff
@@ -37,4 +36,52 @@ public Payment(){
     this.id = UUID.randomUUID().toString();
     this.updated = LocalDateTime.now();
   }
+
+  public String getId() {
```

Comment:
Getters and Setters are provided by Lombok `@Data` annotation already on this model.

Reply:
Ah. Okay. I haven't used Lombok before and wasn't sure how that worked. I added Getters/setters incase.